install/kubernetes/helm/istio/charts/sidecarInjectorWebhook/values.yaml

```diff
@@ -42,3 +42,18 @@ rewriteAppHTTPProbe: false
 neverInjectSelector: []
 
 alwaysInjectSelector: []
+
+# Enable objectSelector to filter out pods with no need for sidecar before calling istio-sidecar-injector.
+# It is disabled by default since this function will only work after k8s v1.15.
+objectSelector:
+  enabled: false
+  autoInject: true
+  autoInjectSelector:
+    matchExpressions:
+    - key: "sidecar.istio.io/inject"
+      operator: NotIn
+      values:
+      - "false"
+  specificInjectSelector:
```

Comment:
Any suggestion for this name? I think `specificInjectSelector ` maybe not a good name.

Reply:
seems like we shouldn't even make this configurable? any reason we should?

Reply:
User may disable sidecar inject by default and specific which pod should inject sidecar. 
This option is aligned with [InjectionPolicyDisabled](https://github.com/istio/istio/blob/2f126947ac1f9ca6d418b2e669aa227198775c3b/pkg/kube/inject/inject.go#L113)

Reply:
I think @howardjohn means that we should not let the end users to change the content of `matchLabels`, because currently we have lots of code are referring to `sidecar.istio.io/inject`.

Reply:
@howardjohn @morvencao fixed